src/commands/moderation/prefixCommands/functions/deleteCategory.ts

```diff
@@ -0,0 +1,93 @@
+import { ChatInputCommandInteraction, Colors, TextChannel, User } from 'discord.js';
+import { constantsConfig, getConn, PrefixCommandCategory, Logger, makeEmbed, clearSinglePrefixCommandCategoryCache } from '../../../../lib';
+
+const noConnEmbed = makeEmbed({
+    title: 'Prefix Commands - Delete Category - No Connection',
+    description: 'Could not connect to the database. Unable to delete the prefix command category.',
+    color: Colors.Red,
+});
+
+const failedEmbed = (categoryId: string) => makeEmbed({
+    title: 'Prefix Commands - Delete Category - Failed',
+    description: `Failed to delete the prefix command category with id ${categoryId}.`,
+    color: Colors.Red,
+});
+
+const doesNotExistsEmbed = (category: string) => makeEmbed({
+    title: 'Prefix Commands - Delete Category - Does not exist',
+    description: `The prefix command category ${category} does not exists. Can not delete it.`,
```

Comment:
```suggestion
    description: `The prefix command category ${category} does not exists. Cannot delete it.`,
```
"Cannot" is generally more common. I suggest we use that but obviously that's personal preference and I'm happy to leave it as is!